Headers/Foundation/NSGeometry.h

```diff
@@ -97,8 +97,8 @@ typedef NSSize *NSSizePointer;
 }</example>
 
  <p>Rectangle.</p> */
-typedef struct _NSRect NSRect;
-struct _NSRect
+typedef struct CGRect NSRect;
+struct CGRect
```

Comment:
Since we’re now defining `struct CGPoint/CGSize/CGRect`, can we please also add the following typedefs to match `CFCGTypes.h` from Apple’s CoreFoundation:

```c
typedef struct CGPoint CGPoint;
typedef struct CGSize CGSize;
typedef struct CGRect CGRect;
```

Reply:
I still don't like the idea that we do this at all, because it will lead to redefinitions.

Reply:
I moved all CG definitions in a CoreFoundation header file. It is not ideal that we install this as part of libs-base, but I cannot think of a better way to do this without major project restructuring.